packages/cli/src/rpc/search/search.ts

```diff
@@ -50,8 +69,12 @@ export async function handler(
     };
     const eventsSearchResponse = await searchSingleAppMap(result.appmap, query, searchOptions);
     results.push({
-      appmap: result.appmap,
-      events: eventsSearchResponse.results,
+      appmap: isAbsolute(result.appmap) ? result.appmap : join(result.directory, result.appmap),
```

Comment:
Always report `appmap` as absolute path. This will make it easier in other systems to resolve it to a file.

Reply:
Is this going to break in like a dev container scenario? I guess we can refit later if needed.